src/_assets/stylesheets/main.scss

```diff
@@ -758,6 +758,7 @@ main .content {
   text-align: center;
   h1, h2, h3, h4, h5, h6 {
     margin-top: 0;
+    margin-bottom: 1em;
```

Comment:
I wouldn't recommend making this style change for all heading levels inside a `card`-styled element. This impacts pages other than the front page (e.g., Languages overview).

The source of the problem is that the h4 element is assigned a margin-bottom of 0 globally (that shouldn't have been done).

Anyhow, see below: I suggest using `<h3>` elements instead (because they look better, and that will allow us to fix the mis-styling of `<h4>` elements later).

Reply:
I wanted the font size of the card headings to match the font size of the Get Started button. I'll try fixing the `<h4>` styling, since that's bothered me for a while.

Reply:
> I wanted the font size of the card headings to match the font size of the Get Started 

I just checked and it is the same size when the heading is `<h3>`.

Reply:
I see the h4 ("Library tour" in the picture below) as 21px and the h3 ("Dart packages" below) as 24px. The button is actually 20px, but I couldn't see the difference.

![image](https://user-images.githubusercontent.com/2164483/35460351-995eed90-0298-11e8-9d87-23828a3174f3.png)